sis-local/README.md

```diff
@@ -0,0 +1,63 @@
+# sis-local
+
+Emulate SiS functionality in a local environment. This may not work perfectly, but it should catch many common errors
+and provide a much smoother experience doing local development and porting to SiS!
+
+- Throw warnings for commands that don't work in SiS
+- (optional) Handles local Snowpark session setup
```

Comment:
I would perhaps vote to focus on the Streamlit-in-Snowflake local experience here, and not care too much about Snowpark session setup just per separation of concern. If Snowpark session setup is an identified pain, then perhaps that's worth another directory / README / doc?

Reply:
Or perhaps do you **need** to Snowpark somewhere to understand if you're in SiS / local?

Reply:
Yes, I think it's the easiest way to distinguish between local development and being in SiS, and I think it's reasonable to package both of those together (though my tendency has been to try and isolate individual components with this repo).

Reply:
I reworked it from the feedback below so it's more clearly separated - LMK what you think